src/main/zapHomeFiles/hud/display.js

```diff
@@ -339,6 +339,40 @@ Vue.component('simple-menu-modal', {
 	}
 })
 
+Vue.component('adv-menu-modal', {
+	template: '#adv-menu-modal-template',
+	props: ['show', 'title'],
+	methods: {
+		close: function() {
+			this.$emit('close');
+		},
+		itemSelect: function(itemId) {
+			this.port.postMessage({'action': 'itemSelected', 'id': itemId});
```

Comment:
you'll need: 			

```
app.keepShowing = true;
app.isAdvMenuModalShown = false;
```
Inserted here

Reply:
This will fix the issue I noticed

Reply:
Weird, I couldnt reproduce the issue, either with Firefox or chrome.
I've added the lines you suggested as they dont seem to cause any problems here, but it would be good to know why you hit this problem while no one else did...